test/google_crawler/keywords_test.exs

```diff
@@ -86,6 +102,57 @@ defmodule GoogleCrawler.KeywordsTest do
     end
   end
 
+  describe "list_keywords/1 with result count filtered" do
+    test "filters keywords with search results more than the given amount if given > in operation" do
+      user = insert(:user)
+      keyword1 = insert(:keyword, user: user)
+      _keyword2 = insert(:keyword, user: user)
+      insert(:search_result, url: "example1.com", keyword: keyword1)
+      insert(:search_result, url: "example2.net", keyword: keyword1)
+
+      [keyword1_in_db] = Keywords.list_keywords(user.id, %{"result_count_operation" => ">", "result_count_value" => "1"})
+
+      assert keyword1_in_db.id == keyword1.id
+    end
+
+    test "filters keywords with search results less than the given amount if given < in operation" do
+      user = insert(:user)
+      keyword1 = insert(:keyword, user: user)
+      keyword2 = insert(:keyword, user: user)
+      insert(:search_result, url: "example1.com", keyword: keyword1)
+      insert(:search_result, url: "example2.net", keyword: keyword1)
```

Comment:
should we create only 1 search result for this case as the test is `< 1` so creating only 1 search result could make the test more accurate

Reply:
Good idea 👍